tests/Type/WebMozartAssert/ImpossibleCheckTypeMethodCallRuleTest.php

```diff
@@ -18,6 +18,8 @@ protected function getRule(): Rule
 
 	public function testExtension(): void
 	{
+		$this->analyse([__DIR__ . '/data/bug-85.php'], []);
```

Comment:
Please add this in a separate test case. The difference is that when bug-85.php fails, we'll also know about failures from impossible-check.php.